centinel/client.py

```diff
@@ -163,7 +199,7 @@ def run(self, data_dir=None):
                     logging.info("tcpdump stopped.")
                     try:
                         pcap_file_name = "pcap_%s-%s.pcap.bz2" % (name, 
-                            datetime.now().isoformat())
+                            exp_start_time)
```

Comment:
this is the perfect use case for a new variable or you could force the start of the arguments to the next line with something like:
("start of line goes here"
 "" % (args))